build/NuSpecs/Intellisense/Microsoft.UI.Xaml.xml

```diff
@@ -1146,12 +1130,27 @@
       <summary>Gets the index of the item the element was prepared for.</summary>
       <returns>The index of the item the element was prepared for.</returns>
     </member>
-    <member name="T:Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost" />
+    <member name="T:Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost">
+      <summary>A helper to coordinate interaction between the <see cref="Microsoft.UI.Xaml.Controls.ItemsRepeater?text=ItemsRepeater" /> and ScrollViewer.
```

Comment:
How about replacing ScrollViewer with Windows.UI.Xaml.Controls.ScrollViewer?  The ItemsRepeaterScrollHost is not needed with the MUX ScrollViewer.